### PR TITLE
[GH-545] added aeternity erlang compatibility doc

### LINKED ISSUE #545: Write step-by-step guide on how to check consensus to be epoch 0.16 compatible
add a docs/epoch-compatibility.md with `## epoch 0.16` headline and description

### PR DESCRIPTION
**Pull request by thepiwo**
resolves #545

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 # Aeternity Elixir Full Node
 This is an elixir full node implementation of the aeternity specification.
 
+Compatibility to the erlang aeternity implementation is documented in [docs/aeternity-erlang-compatibility.md](docs/aeternity-erlang-compatibility.md).
```
Comment (grzegorz225) [inline README.md]: I think this should be written more like:

Tests for checking compatibility to the epoch aeternity implementation are documented in...

File: docs/aeternity-erlang-compatibility.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
+# aeternity Erlang implementation compatibility
```
Comment (grzegorz225) [inline docs/aeternity-erlang-compatibility.md]: Shouldn't we make a test framework for this?
We could download epoch via the makefile and create a wrapper for ExUnit to directly invoke and test code running in epoch.

Comment (thepiwo) [reply docs/aeternity-erlang-compatibility.md]: Yes, there is already and issue for this https://github.com/aeternity/elixir-node/issues/574

File: README.md (hunk 1 above)
Comment (thepiwo) [reply README.md]: this should be more of a documentation with future versions just stating changes, without more specific cases. The sample cases are just for outsiders to quickly get up and running to check these the documented cases.

**Pull request merged**